content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md

```diff
@@ -16,8 +18,19 @@ Svelte is a JavaScript front-end framework for building reactive and interactive
   - [Set up the Team component](#set-up-the-team-component)
   - [Set up the Footer component](#set-up-the-footer-component)
 - [Conclusion](#conclusion)
+- [Additional Resources](#additional-resources)
 
 ### Setting up the application with Vite and TypeScript
+Vite is a JavaScript development server tooling that makes front-end development more easier. Vite distributes your app source files through native ES modules, allowing you to quickly start or reload your development server. Vite starts the server right away, and it takes the dependencies that don't change often and pre-bundles them using ES build.
```

Comment:
Rephrase to this - Vite is a JavaScript development server tool that makes front-end development easier.